public/utils/assets.ts

```diff
@@ -1,10 +1,12 @@
 import { ASSETS_BASE_URL_PREFIX } from '../../common/constants';
 import { getUiSettings } from '../kibana-services';
 
-export const getAssetURL = (assetURL: string) => `${ASSETS_BASE_URL_PREFIX}${assetURL}`;
+export const getAssetURL = (assetURL: string) =>
+  `${ASSETS_BASE_URL_PREFIX}${assetURL}`;
 
 export const getThemeAssetURL = (asset: string, theme?: string) => {
   theme = theme || (getUiSettings()?.get('theme:darkMode') ? 'dark' : 'light');
+  console.log(`Theme: ${theme}`);
```

Comment:
This console.log() should be deleted, right?

Reply:
Removed in [96e1fd2](https://github.com/wazuh/wazuh-kibana-app/pull/5620/commits/96e1fd2446105b1c162a5a177f9d196e4ecd6d70)